go.mod

```diff
@@ -0,0 +1,46 @@
+module github.com/networkservicemesh/integration-k8s-kind/tests
```

Comment:
Why the /tests suffix that doesn't correspond to the actual repo name?

Reply:
Thanks! Fixed.